pkg/apis/serving/v1alpha2/inferenceservice_types.go

```diff
@@ -60,6 +60,10 @@ type DeploymentSpec struct {
 	// This is the up bound for autoscaler to scale to
 	// +optional
 	MaxReplicas int `json:"maxReplicas,omitempty"`
+	// Parallelism specifies how many requests can be processed concurrently, this sets the target
+	// concurrency for autoscaling and for python model server it also sets the number of workers
+	// +optional
+	Parallelism int `json:"parallelism,omitempty"`
```

Comment:
Can you specify what parallelism of 0 means? Also, can we remove any concept of python. "Model Servers that support concurrency will use this value"